package.json

```diff
@@ -9,7 +9,8 @@
     "scripts": {
         "build": "yarn build:libraries && yarn build:subgraph && yarn build:website && yarn build:docs",
         "build:libraries": "yarn workspaces foreach -Apt --no-private run build",
-        "build:subgraph": "yarn workspace semaphore-subgraph build:sepolia",
+        "clean:subgraph": "rimraf apps/subgraph/node_modules",
+        "build:subgraph": "yarn clean:subgraph && yarn workspace semaphore-subgraph build:sepolia",
```

Comment:
I think this issue is solved with these two lines. Why it is necessary to add `clean:branch` and update the workspaces object?

Reply:
i think it was a testing error as i checking for test cases - clean:branch  is not needed at all.

Reply:
hi @vplasencia, let me know if there is any update.